packages/lodestar-beacon-state-transition/src/util/attestation.ts

```diff
@@ -138,22 +138,3 @@ export function isUnaggregatedAttestation(attestation: Attestation): boolean {
   }
   return count === 1;
 }
-
-/**
- * Compute the correct subnet for an attestation for Phase 0.
- */
-export function computeSubnetForAttestation
```

Comment:
We should keep the slow versions for now.

Maybe we eventually rm -r all of the slow implementation?

Reply:
Well they aren't used in state transition so maybe we should pull it out allthogether?

Reply:
My current thought is eventually we could split state transition into two packages, normal and fast (like eth2spec and eth2fastspec python libraries). Slow implementations usually look like `fn(config, state, a,b,c)`, while fast looks like `epochCtx.fn(a, b,c)` or `epochCtx.fn(state, a, b, c)` depending if state is needed (in most cases it is not). We can re-export some low-level utils from slow in the fast package. And lodestar could import solely from fast.

These attestation helpers are a weird case, since they're not used directly by state transition, but still require committee info. Imo we leave them in state transition as state transition "helpers/misc utilities".